packages/bs-reform/src/ReFormNext.re

```diff
@@ -127,6 +147,7 @@ module Make = (Config: Config) => {
     arrayRemoveBy: 'a. (Config.field(array('a)), 'a => bool) => unit,
     submit: unit => unit,
     resetForm: unit => unit,
+    setInitialState: Config.state => unit,
```

Comment:
I think we can actually create a helper called `setValues`

Reply:
i think that setValues means set all values passed in, and setInitialState means won't set the value when its fieldState is not Pristine, maybe we can call it setinitialValues

Reply:
that makes sense, I'll take a better look soon so we can get it merged

Reply:
hello @fakenickels would you have time to merge this

Reply:
Sorry about the delay, the past few days have we have been a bit busy. I'll gonna finish the review today and give you a better feedback soon